app/grandchallenge/evaluation/forms.py

```diff
@@ -211,11 +211,15 @@ class MethodForm(ContainerImageForm):
         help_text="Which phase is this evaluation container for?",
     )
 
-    def __init__(self, *args, challenge, **kwargs):
+    def __init__(self, *args, challenge, phase, **kwargs):
         super().__init__(*args, **kwargs)
 
         self.fields["phase"].queryset = challenge.phase_set.all()
 
+        if phase:
+            self.fields["phase"].initial = phase
+            self.fields["phase"].widget = HiddenInput()
```

Comment:
Is `challenge` still necessary? `phase` should always be set now so the if statement should not be necessary.

Reply:
Depends on whether we keep the unfiltered view working or not. At the moment, `phase` can be unset if someone navigates to `/methods/create/` with `?phase=foo` added to the url. I kept that working intentionally for admins who might have bookmarked these urls (same goes for the SubmissionList and EvalList views, they still work without `phase` set). If we don't want to offer the unfiltered views anymore, than the if statement here (and in some other places) is not necessary indeed.

Reply:
Lets remove the unfiltered views as we're not linking to them.

Reply:
For the existing pages add a temporary redirect to the first non-hidden phase for the challenge.